client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -155,15 +155,10 @@ export class PluginAutoUpdateToggle extends Component {
 				'DISABLE_AUTOUPDATE_PLUGIN',
 			] ),
 			getDisabledInfo = this.getDisabledInfo(),
-			label = getDisabledInfo
-				? translate( 'Autoupdates disabled', {
-						context:
-							'this goes next to an icon that displays if the plugin has "autoupdates disabled" active',
-					} )
-				: translate( 'Autoupdates', {
-						context:
-							'this goes next to an icon that displays if the plugin has "autoupdates" active',
-					} );
+			label = translate( 'Autoupdates', {
```

Comment:
:disappointed: 44 existing translations will be lost with this change.
If the string must change, the following option has already been translated **28** times:
`translate( 'Autoupdates disabled', { context: 'this goes next to an icon that displays if the plugin has "autoupdates disabled" active'} )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plugin-action-styles)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Hmm, not sure which 44 translations are tied to this string but since we are dropping the "disabled" part (the UI will now signify that it's disabled in a better way so no need for additional text/explanation), I think it's okay. Or not?